fabric/bin/lib/common_ledger.sh

```diff
@@ -111,7 +111,12 @@ ledger_init() {
     try mkdir -p ${FABRIC_STATE_DIR}
     try rm -rf ${FABRIC_STATE_DIR}/*
 
+    # this part would be used for raft
+    #try ${FABRIC_BIN_DIR}/cryptogen generate --config=./crypto-config.yaml --output="organizations" 
     # 2. start orderer
+    # - Creating a genesisblock for orderer-system-channel......."
+    try ${CONFIGTXGEN_CMD} -outputBlock genesisblock -profile SampleDevModeSolo -channelID orderer-system-channel
```

Comment:
Actually, realized only now that this command is also an addition compared to v1.4. The issue right now is that this generates a file (genesisblock) in the repo. So we should ever add that to .gitignore or maybe create it in `/tmp/hyperledger/test`  (where we probably should put ercc.tar.gz, see other comment on temporary files). In the latter case we also would have to probably change the `BootstrapFile` stanza in `orderer.yml`